docs/getting-started/installation.md

```diff
@@ -90,67 +133,71 @@ Watch the `pip install` output messages for the `Scripts` folder path on your sy
+Compatible versions include Ubuntu 22.04 LTS and Ubuntu 20.04 LTS.
 
 Alternatively, you can [install SQLite on Red Hat Custom Linux (RHEL)](#install-sqlite-on-rhel) or use the `conda` virtual environment manager and configure a compatible SQLite version.
 
 ## Using a self-signed SSL certificate
 
-If you're using a self-signed SSL certificate, you need to configure your
-environment to trust the certificate. You can add the
-certificate to your system bundle and pointing your tools to use that bundle by configuring the `SSL_CERT_FILE` environment variable.
+If you're using a self-signed SSL certificate, you need to configure your environment to trust the certificate.
+You can add the certificate to your system bundle and pointing your tools to use that bundle by configuring the `SSL_CERT_FILE` environment variable.
 
 If the certificate is not part of your system bundle, you can set the
 `PREFECT_API_TLS_INSECURE_SKIP_VERIFY` to `True` to disable certificate verification altogether.
 
 ***Note:*** Disabling certificate validation is insecure and only suggested as an option for testing!
 
-## Installing for development
+## Proxies
 
-If you'd like to install a version of Prefect for development:
+Prefect supports communicating via proxies through environment variables.
+Whether you are using Prefect Cloud or hosting your own Prefect server instance, set `HTTPS_PROXY` and `SSL_CERT_FILE` in your environment, and the underlying network libraries will route Prefect’s requests appropriately.
 
-1. Clone the [Prefect repository](https://github.com/PrefectHQ/prefect).
-2. Install an editable version of the Python package with `pip install -e`.
-3. Install pre-commit hooks.
+Alternatively, the Prefect library will connect to the API via any proxies you have listed in the `HTTP_PROXY` or `ALL_PROXY` environment variables.  
+You may also use the `NO_PROXY` environment variable to specify which hosts should not be sent through the proxy.
+
+For more information about these environment variables, see the [cURL documentation](https://everything.curl.dev/usingcurl/proxies/env).
+
+## `prefect-client` library
+
+The `prefect-client` library is a minimal installation of Prefect designed for interacting with Prefect Cloud or a remote self-hosted server instance.
+
+`prefect-client` contains a subset of functionality with a smaller installation size, making it ideal for use in lightweight, resource-constrained, or ephemeral environments.
+It omits all CLI and server components found in the `prefect` library.
+
+Install the latest version with:
 
 <div class="terminal">
 ```bash
-$ git clone https://github.com/PrefectHQ/prefect.git
-$ cd prefect
-$ pip install -e ".[dev]"
-$ pre-commit install
+pip install -U prefect-client
 ```
 </div>
 
-See the [Contributing](/contributing/overview/) guide for more details about standards and practices for contributing to Prefect.
+## Additional Prefect requirements
```

Comment:
Why have a whole section on additional requirements if the other item in that section is SQLite?